addons/account/tests/test_invoice_tax_totals.py

```diff
@@ -794,6 +795,13 @@ def test_cash_rounding_amount_total_rounded(self):
             self.assertEqual(move.tax_totals['amount_total'], 544.7)
             self.assertEqual(move.tax_totals['amount_total_rounded'], 545)
 
+            moves += move
+
+        # if rounding_method is changed then rounding shouldn't be recomputed in posted invoices
+        moves.action_post()
+        cash_rounding_add_invoice_line.rounding_method = 'DOWN'
+        moves.to_check = True
```

Comment:
I'd prefer to have a separated test for this.

* create a rounding method
* create a simple invoice with amounts that would be rounded differently
* post it
* update the rounding method
* change `to_check` (or any field that shouldn't change the values of the invoice)

It won't be much longer but it will be way easier to understand and maintain